server/README.md

```diff
@@ -1,26 +1,6 @@
-<p align="center">
```

Comment:
Move the info from this README to the main README of the project. Then, this README can be deleted